### PR TITLE
Improve @/climatemappedafrica `HURUmap/Panel`

### LINKED ISSUE #926: ClimateMapped: Improve data page navigation labels
Improve the labelling on Navigations in the explore page in ClimateMapped

Request Link: [Feature 6](https://docs.google.com/document/d/1TUrdCRnbj6TSqiuwITKUpnDUEi_ktm_3PmgsaD0y2bw/edit#heading=h.9ih8c5e33n7n)

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

This PR improves the `HURUmap/Panel`:

 - [x] Desktop
   - [x] Ensure `Drawer` acts like a drawer.
   - [x] Ensure `Drawer` buttons show tooltip.
   - [x] Ensure (top) alignment of drawer buttons, location highlight and legend.
   - [x] Ensure `Legend` resembles location highlight i.e. slightly transparent, rounded borders, etc.
   - [x] Ensure `Legend` value/colour and text vertically aligns,
   - [x] etc.
 - [x] Mobile
   - [x] Ensure tabs don't carry over the blue of links

The PR also migrates (where applicable) styles in `HURUmap/Panel` and related components to `sx` from `useStyles`.

Fixes #926 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### Desktop

D1

D2

### Mobile

M

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+function DesktopPanel({ sx, ...props }) {
+  const [value, setValue] = useState();
+  const { isOpen: tutorialOpen } = useTour();
+
+  const closeDrawer = () => {
+    setValue(undefined);
+  };
+  const handleValueChange = useCallback((newValue) => {
+    console.log("BOOM H: ", newValue);
```
Comment (koechkevin) [inline apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js:17]:
```suggestion
```

**Review by kelvinkipruto — APPROVED** [review]
I haven't run it yet but the code looks good @kilemensi. Just curious, to get it rendered, did you restore the `explore` page deleted in #945

**Review by koechkevin — APPROVED** [review]

File: apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js (hunk 1 above)
Comment (kilemensi) [reply apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js:17]: I love making things go boOM!

**Comment by kilemensi** [comment]
>I haven't run it yet but the code looks good @kilemensi. Just curious, to get it rendered, did you restore the explore page deleted in https://github.com/CodeForAfrica/ui/pull/945

🤔 ... don't think I restored it @kelvinkipruto. It could be that I hadn't delete the page at the time I took the screenshots.

**Comment by kelvinkipruto** [comment]
@kilemensi Any other changes to this, or we can merge it?

**Pull request merged**